projects/aca-shared/src/lib/services/app.extension.service.ts

```diff
@@ -350,6 +350,10 @@ export class AppExtensionService implements RuleContext {
     };
   }
 
+  updateSidebarActions() {
```

Comment:
missing unit tests for this function

Reply:
Done

Reply:
@rbahirsheth this is done not correctly. Now there is added following test: 
it('should update the sidebarActions', () => {
    spyOn(service, 'updateSidebarActions').and.callThrough();
    service.updateSidebarActions();
    expect(service.updateSidebarActions).toHaveBeenCalled();
  });

This test tests nothing because you call updateSidebarActions function and then in expect you check if that function is called which you just called manually... you should test what that function does - if everything inside that function is called properly

Reply:
Done